### PR TITLE
Added documentation job attribute for all .md files

### PR DESCRIPTION
**Pull request by nignatovsky**
Documentation job triggering on changes to any documentation files

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -36,9 +36,9 @@
...
 /demos/**/*.md -omz.ci.job-for-change.demos
 
-/models/**/*.md omz.ci.job-for-change.documentation
-
 /tools/accuracy_checker/** omz.ci.job-for-change.ac
 /tools/accuracy_checker/configs/*.yml -omz.ci.job-for-change.ac
 /tools/downloader/** omz.ci.job-for-change.downloader
 /tools/**/*.md -omz.ci.job-for-change.ac -omz.ci.job-for-change.downloader
+
+**/*.md omz.ci.job-for-change.documentation
```
Comment (IRDonch) [inline .gitattributes:44]:
```suggestion
*.md omz.ci.job-for-change.documentation
```

`**/` in front is redundant.

I also suggest to merge this rule with the other rules for Markdown files (i.e., disable all non-documentation jobs for changes to `*.md`).

Comment (nignatovsky) [reply .gitattributes]: Done.

Comment (IRDonch) [reply .gitattributes]: No you haven't. You need to disable the other jobs for changes to `*.md`.

Comment (nignatovsky) [reply .gitattributes]: Oops, I didn't notice "-" sign; fixed.

**Review by IRDonch — APPROVED** [review]

**Comment by IRDonch** [comment]
I merged the corresponding CI pipeline change, so I'm removing the draft status.

**Comment by IRDonch** [comment]
Jenkins: test this please

**Pull request merged**